<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmNewTexArray</class>
 <widget class="QDialog" name="frmNewTexArray">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>352</width>
    <height>70</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>352</width>
    <height>70</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>352</width>
    <height>70</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="btnCancel">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>40</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnOk">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>40</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>331</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1,0,1">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Texture Width: </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="spnWidth">
      <property name="maximum">
       <number>99999</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Texture Height:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="spnHeight">
      <property name="maximum">
       <number>99999</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>spnWidth</tabstop>
  <tabstop>spnHeight</tabstop>
  <tabstop>btnOk</tabstop>
  <tabstop>btnCancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
